### PR TITLE
Changed backgroundcolor + alignment and change textstyle name

### PR DESCRIPTION
**Pull request by Pavelbarrosq**

### CODE REVIEW

File: Src/Assets/HedvigColors.swift [hunk 1]
```diff
@@ -148,7 +148,7 @@ extension UIColor {
     static let darkPink = UIColor(red: 0.67, green: 0.0, blue: 0.27, alpha: 1.0)
     static let yellow = UIColor(red: 1.00, green: 0.80, blue: 0.30, alpha: 1.0)
     static let grass500 = UIColor(red: 0.0, green: 0.56, blue: 0.45, alpha: 1.0)
-    static let sunflower300 = UIColor(red: 251.0 / 255.0, green: 227.0 / 255.0, blue: 86.0 / 255.0, alpha: 1.0)
+    static let sunflower300 = UIColor(red: 248.0 / 255.0, green: 253.0 / 255.0, blue: 137.0 / 255.0, alpha: 1.0)
```
Comment (sampettersson) [inline Src/Assets/HedvigColors.swift:151]: maybe we should just call this yellow and remove old yellow? 🤔

File: Src/Screens/ProfileScreen/Screens/MyPayment/Viewables/HeaderCard/PaymentHeaderNextCharge.swift [hunk 2]
```diff
@@ -48,6 +51,7 @@ extension PaymentHeaderNextCharge: Viewable {
                 view.backgroundColor = .primaryBackground
             } else {
                 label.value = String(key: .PAYMENTS_CARD_NO_STARTDATE)
+                label.textColor = .darkGray
```
Comment (sampettersson) [inline Src/Screens/ProfileScreen/Screens/MyPayment/Viewables/HeaderCard/PaymentHeaderNextCharge.swift:54]: Using textColor directly is prohibited, this comes directly from the textstyle instead.

File: Src/Screens/ProfileScreen/Screens/MyPayment/Viewables/HeaderCard/PaymentHeaderNextCharge.swift [hunk 3]
```diff
@@ -27,11 +27,14 @@ extension PaymentHeaderNextCharge: Viewable {
         contentContainer.distribution = .equalSpacing
         view.addSubview(contentContainer)
 
-        let label = UILabel(value: "", style: TextStyle.body.zeroedLineSpacing)
+        let label = UILabel(value: "", style: .bodyText)
         contentContainer.addArrangedSubview(label)
 
         contentContainer.snp.makeConstraints { make in
-            make.top.bottom.leading.trailing.equalToSuperview()
+            make.left.equalToSuperview().offset(10)
```
Comment (sampettersson) [inline Src/Screens/ProfileScreen/Screens/MyPayment/Viewables/HeaderCard/PaymentHeaderNextCharge.swift:34]: Couldn't we do this with layout margins instead?

File: Src/Generic/Styling/TextStyle.swift [hunk 4]
```diff
@@ -47,7 +47,7 @@ extension TextStyle {
         style.lineSpacing = 6
     }
 
-    static let bodyButtonText = TextStyle.default.restyled { (style: inout TextStyle) in
+    static let bodyText = TextStyle.default.restyled { (style: inout TextStyle) in
```
Comment (sampettersson) [inline Src/Generic/Styling/TextStyle.swift:50]: I think remove this style and use `bodySmallSmall` instead.

Comment (Pavelbarrosq) [reply Src/Generic/Styling/TextStyle.swift:50]: Went back as it was and put back bodyButtonText so i didn't have to change something in offerscreen change startdate button. So i just updated the payment style to bodySmallSmall :)

**Pull request closed without merging**